testgrid/cmd/updater/main.go

```diff
@@ -119,62 +225,108 @@ type JunitResult struct {
 	Name      string  `xml:"name,attr"`
 	Time      float64 `xml:"time,attr"`
 	ClassName string  `xml:"classname,attr"`
-	Failure   *string `xml:"failure"`
-	Output    *string `xml:"system-out"`
-	Skipped   *string `xml:"skipped"`
+	Failure   *string `xml:"failure,omitempty"`
+	Output    *string `xml:"system-out,omitempty"`
+	Error     *string `xml:"system-err,omitempty"`
+	Skipped   *string `xml:"skipped,omitempty"`
 }
 
-func (jr JunitResult) RowResult() state.Row_Result {
+func (jr JunitResult) Message() string {
+	const max = 140
+	var msg string
+	switch {
+	case jr.Failure != nil && *jr.Failure != "":
+		msg = *jr.Failure
+	case jr.Skipped != nil && *jr.Skipped != "":
+		msg = *jr.Skipped
+	case jr.Output != nil && *jr.Output != "":
+		msg = *jr.Output
+	}
+	if l := len(msg); max == 0 || l <= max {
```

Comment:
`if max == 0 || len(msg) <= max {`

Reply:
`l` is used in the `else` statement (`msg[l-h-1:]`)